lib/sql.js

```diff
@@ -2,19 +2,29 @@ var util = require('util');
 var async = require('async');
 var assert = require('assert');
 var Connector = require('./connector');
+var debug = require('debug')('loopback:connector:sql');
+var ParameterizedSQL = require('./parameterized-sql');
 
 module.exports = SqlConnector;
 
 /**
- * Base class for connectors that are backed by relational databases/SQL
+ * Base class for connectors that connect to relational databases using SQL
  * @class
  */
 function SqlConnector() {
+  // Call the super constructor
   Connector.apply(this, [].slice.call(arguments));
 }
 
+// Inherit from the base Connector
 util.inherits(SqlConnector, Connector);
 
+// Export ParameterizedSQL
+SqlConnector.ParameterizedSQL = ParameterizedSQL;
```

Comment:
Having said that, I think we don't have any guideline for names containing abbreviations. .NET uses pascal/camel case `Sql`, I guess Java world prefers all-upper-case spelling `SQL`? I don't want to impose my naming preferences on you, my only goal is to keep names consistent.

If you prefer `SQL` over `Sql`, then perhaps this is the right time to change the class name to `SQLConnector`? The patch is going be released as a major version, so it's ok to make breaking changes. To lessen the impact of the rename, we can easily provide an alias in the index file: `exports.SqlConnector = exports.SQLConnector`.